presto-main/src/main/java/com/facebook/presto/operator/JoinOperatorInfo.java

```diff
@@ -29,29 +29,32 @@
     private final JoinType joinType;
     private final long[] logHistogramProbes;
     private final long[] logHistogramOutput;
+    private final long lookupSourcePositions;
 
-    public static JoinOperatorInfo createJoinOperatorInfo(JoinType joinType, long[] logHistogramCounters)
+    public static JoinOperatorInfo createJoinOperatorInfo(JoinType joinType, long[] logHistogramCounters, long lookupSideRows)
     {
         long[] logHistogramProbes = new long[HISTOGRAM_BUCKETS];
         long[] logHistogramOutput = new long[HISTOGRAM_BUCKETS];
         for (int i = 0; i < HISTOGRAM_BUCKETS; i++) {
             logHistogramProbes[i] = logHistogramCounters[2 * i];
             logHistogramOutput[i] = logHistogramCounters[2 * i + 1];
         }
-        return new JoinOperatorInfo(joinType, logHistogramProbes, logHistogramOutput);
+        return new JoinOperatorInfo(joinType, logHistogramProbes, logHistogramOutput, lookupSideRows);
```

Comment:
I'd name this `lookupSourcePositions` to be consistent with the other constructors.